app/src/main/java/com/alexbar/layoutbasic/ui/theme/Dimens.kt

```diff
@@ -0,0 +1,49 @@
+package com.alexbar.layoutbasic.ui.theme
+
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+object Dimens {
+    val content_screen_spacer_height_100 = 100.dp
+    val content_screen_spacer_height_80 = 80.dp
+    val content_screen_spacer_height_36 = 36.dp
+    val content_screen_box_padding_24 = 24.dp
+    const val content_screen_spacer_weight_1 = 1f
+
+    val playing_card_height = 250.dp
+    val playing_card_width = 200.dp
+    val playing_card_corner_shape = 28.dp
+    val playing_card_content_box_padding_16 = 16.dp
+    val playing_card_content_font_size_12 = 12.sp
+    val playing_card_content_font_size_14 = 14.sp
+    val playing_card_content_spacer_18 = 18.dp
+    const val playing_card_gradient_start_y = 300f
+
+    val list_songs_spacer_16 = 16.dp
+    val list_songs_item_image_size_50 = 50.dp
+    val list_songs_item_image_corner_shape = 8.dp
+    val list_songs_item_spacer_18 = 18.dp
+    val list_songs_item_more_icon_size = 20.dp
+    val list_songs_item_padding_horizontal_16 = 16.dp
+    val list_songs_item_padding_vertical_8 = 8.dp
+    val list_songs_item_font_size_12 = 12.sp
+    val list_songs_item_font_size_14 = 14.sp
+    const val list_songs_item_weight_1 = 1f
```

Comment:
@ezatpanah Is it appropriate to put all these types of values into a single file? I've organized values based on the specific components they are utilized in, but I've noticed some values are duplicated across multiple components. Would it be more efficient to define a common value and apply it universally across all components that require this specific value?